example/android/app/src/main/AndroidManifest.xml

```diff
@@ -7,12 +7,14 @@
       android:name=".MainApplication"
       android:label="@string/app_name"
       android:icon="@mipmap/ic_launcher"
+      android:largeHeap="true"
```

Comment:
Why is this needed?

Reply:
For handle large base64 image (so long string)